docs/topics/testing/tools.txt

```diff
@@ -425,6 +425,20 @@ Specifically, a ``Response`` object has the following attributes:
             >>> response.context['name']
             'Arthur'
 
+    .. method:: json()
+
+        .. versionadded:: 1.9
+
+        The body of the response, parsed as JSON. Extra keyword arguments are
+        passed to ``json.loads``, see the `Python documentation
+        <https://docs.python.org/3/library/json.html#json.loads>`_ for more
+        information. If the Content-Type is not `"application/json"`
```

Comment:
add backtickets around Content-Type
need to double backticks for application/json to have an effect on the rendered output